raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -58,7 +58,12 @@ library ChannelManagerLibrary {
 
         address[] storage existingChannels = self.nodeChannels[msg.sender];
```

Comment:
@czepluch , @schmir is right. You are iterating an array and attempting to modify it during the iteration. That will not end up nicely.